### PR TITLE
resource/alicloud_ess_scaling_group: support composable and add az_balance,allocation_strategy  and spot_allocation_strategy.

### PR DESCRIPTION
**Pull request by fuliu-zln**
…ocation_strategy  and spot_allocation_strategy.

### CODE REVIEW

File: alicloud/resource_alicloud_ess_scaling_group.go [hunk 1]
```diff
@@ -113,9 +113,26 @@ func resourceAlicloudEssScalingGroup() *schema.Resource {
...
 				Optional:     true,
 				Default:      "PRIORITY",
-				ValidateFunc: StringInSlice([]string{"PRIORITY", "BALANCE", "COST_OPTIMIZED"}, false),
+				ValidateFunc: StringInSlice([]string{"PRIORITY", "BALANCE", "COST_OPTIMIZED", "COMPOSABLE"}, false),
 				ForceNew:     true,
 			},
+			"az_balance": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_ess_scaling_group.go:122]: 这个字段是不是可以改成false的，如果可以，去掉Computed

Comment (fuliu-zln) [reply alicloud/resource_alicloud_ess_scaling_group.go:122]: 是的 done

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**